core/src/processing/feeder.cc

```diff
@@ -139,6 +139,7 @@ void feeder::run() {
   {
     QWriteLocker lock(&_client_mutex);
     _client.clear();
+    _subscriber.remove_queue_files();
```

Comment:
Instead of using a subscriber's method to reach the muxer, could you  use _subscriber.get_muxer().remove_queue_files() to be consistent with the codebase ?

As a consequence, remove_queue_files() could be removed from the subscriber class.